### ISSUE TITLE
[Accessbility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Link a Cluster-Link a Cluster_HD Insight_Screen Reader: JAWS is not reading the labels present in 'Link a cluster' Pop up

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/azure-tools-for-java#3616]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #ScreenReader; #JAWS; #MAS1.3.1

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)
MAS Violated: MAS1.3.1

**Note: Similar issue is observed in**
1.Create Spark Scala Project
2.Link SQL Server big data cluster

**Repro Steps:**
1)Turn on the screen reader (JAWS)
2)Launch IntelliJ IDEA 
3)Tab till Create New Project button and press enter
4)Tab till view-->Tool Window-->Azure Explorer and press enter
5)Tab till signin-->copy and open & sign in to azure
6)Tab till select button and press enter
7)Tab till HDInsight and open context menu by using keyboard key
8)Tab till link a cluster and press enter
9)Verify whether JAWS is reading the labels present in 'Link a cluster' Pop up

**Actual Result:**
JAWS is not reading the labels 'Server', 'Knox port', 'Name', 'Authentication Type' and 'User Name' present in 'Link a cluster' Pop up

**Expected Result:**
JAWS should read the labels present in 'Link a cluster' Pop up

**User Impact:**
Visually disabled users will find difficult, if the screen reader (JAWS) is not reading the labels provided 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug _Jaws not reading the labels in 'Link a cluster'.pptx](https://github.com/microsoft/azure-tools-for-java/files/3646760/Bug._Jaws.not.reading.the.labels.in.Link.a.cluster.pptx)

[Bug _Create Spark Scala Project_Jaws not reading the label name.pptx](https://github.com/microsoft/azure-tools-for-java/files/3646785/Bug._Create.Spark.Scala.Project_Jaws.not.reading.the.label.name.pptx)

[Bug _Link SQL Big Data Cluster_Jaws not reading the labels.pptx](https://github.com/microsoft/azure-tools-for-java/files/3646863/Bug._Link.SQL.Big.Data.Cluster_Jaws.not.reading.the.labels.pptx)

**Comment by t-rufang** [comment]
Note:

Screen reader is not reading "Build tool" when focus is at combo box `Maven`.

Screen Reader is not reading the label when focus is at the text field
![image](https://user-images.githubusercontent.com/32627233/65673077-c22ff480-e07c-11e9-89e0-54ec1537acd8.png)

**Comment by t-rufang** [comment]
Fixed in #3673

**Comment by v-lasiva** [comment]
Regressed the issue in updated version of IntelliJ having the details,
Version: IntelliJ IDEA 2019.2.2

The issue that was mentioned in the note section for "Create spark Scala project", Jaws is still not reading "Build tool" label. Added attachment for reference.

Remaining 2 issues are resolved. Jaws is reading all the labels when "Link a Cluster and Link a Big data server" popups are opened. Added attachments for reference.

[Bug_Jaws is not reading the Build tool label_Issue not fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3851028/Bug_Jaws.is.not.reading.the.Build.tool.label_Issue.not.fixed.pptx)
[Bug_Jaws is reading labels for Link a cluster popup_Issue fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3851029/Bug_Jaws.is.reading.labels.for.Link.a.cluster.popup_Issue.fixed.pptx)
[Bug_Jaws is reading the labels in Link a big data server popup_Issue fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3851030/Bug_Jaws.is.reading.the.labels.in.Link.a.big.data.server.popup_Issue.fixed.pptx)

**Comment by t-rufang** [comment]
Fixed in #3760

**Comment by v-lasiva** [comment]
Regressed the issue in the latest build,

Now Jaws is reading the "Build Tool" Label as expected.
As the issue is resolved, Hence closing the issue
Added attachment foe reference.

[Bug_Jaws is reading the build tool now_Issue fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3868821/Bug_Jaws.is.reading.the.build.tool.now_Issue.fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749716
